src/envs/painting.py

```diff
@@ -535,9 +542,9 @@ def _get_tasks(self, num_tasks: int, num_objs_lst: List[int],
                 state.set(self._robot, "fingers", 0.0)
                 state.set(target_obj, "grasp", grasp)
                 state.set(target_obj, "held", 1.0)
-                if CFG.env == "repeated_nextto_painting":
-                    state.set(target_obj, "pose_y",
-                              state.get(self._robot, "pose_y"))
+                state.set(target_obj, "pose_y",
+                          state.get(self._robot, "pose_y"))
+                if self._update_z_poses:
                     state.set(target_obj, "pose_z",
```

Comment:
just curious what would break if we did this in regular painting too (not just rnt painting)

Reply:
you're right, I think nothing would break. I'll experiment with changing this after the current nightly runs finish (before merging this PR).

Reply:
i think you can kill the master supercloud (your account) right? it looks fine enough

Reply:
I attempted to make this change, but it became a rabbit hole that was leading to changes in unit tests and ground truth nsrts, not to mention I don't know how it would affect nsrt learning and predicate invention. so I'm going to abandon this.

Reply:
kk thx for trying